components/feature/intent/src/main/java/mozilla/components/feature/intent/IntentProcessor.kt

```diff
@@ -141,6 +95,6 @@ class IntentProcessor(
     }
 
     companion object {
-        const val ACTIVE_SESSION_ID = "activeSessionId"
+        const val ACTIVE_SESSION_ID = EXTRA_SESSION_ID
```

Comment:
Do we still need this `ACTIVE_SESSION_ID`?

Reply:
Yes, until we can remove references to it in Fenix.